src/solver/hierarchical-iterative.cc

```diff
@@ -468,24 +449,19 @@ namespace hpp {
           return false;
         }
         size_type iq = iq_ [f];
-        size_type iv = iv_ [f];
         size_type nq = space->nq ();
+#ifndef NDEBUG
```

Comment:
I guess this was unintentionally staged.

Reply:
No, there is an assert a few lines later.

Reply:
Oh, yes. Sorry.